seed/challenges/00-getting-started/getting-started.json

```diff
@@ -908,6 +1064,57 @@
           ""
         ]
+      ],
+      "TitleIt": "Impara Cosa fare se ti trovi in ​​difficoltà",
+      "DescriptionIt": [
+        [
+          "//i.imgur.com/lzKvwU2.jpg",
+          "Il testo \" Programmare è difficile. \"",
+          "Programmare è difficile. Prima o poi rimarrai bloccato. Succede anche i programmatori esperti. La chiave è sapere come uscirne.",
+          ""
+        ],
+        [
+          "//i.imgur.com/sfsidp6.jpg",
+          "Il testo \" Ci vuole tempo per diventare bravi sviluppatori. \"",
+          "Ci vuole tempo per diventare bravi sviluppatori. Non ti aspetteresti di battere un campione di scacchi dopo aver giocato per 3 mesi. Non aspettarti di realizzare il prossimo Facebook dopo aver programmato per 3 mesi.",
+          ""
+        ],
+        [
+          "//i.imgur.com/EoTfOyC.jpg",
+          "Un'immagine di alcuni dei nostri camper che programmano insieme a Montreal.",
+          "Continua ad esercitarti a programmare tutti i giorni e discuti con altre persone che programmano, e potrai diventare uno sviluppatore pronto per il lavoro.",
+          ""
+        ],
+        [
+          "//i.imgur.com/EWWZBag.jpg",
+          "Un'immagine con il testo \" 1. Leggi l'errore 2. Cerca su Google 3. Chiedi aiuto. ",
+          "Ogni volta che ti trovi in ​​difficoltà o non sai cosa fare. Leggi-Cerca-Chiedi",
+          ""
+        ],
+        [
+          "//i.imgur.com/99BfAcK.jpg",
+          "Un'immagine che mostra la documentazione di jQuery",
+          "Innanzitutto, leggi la documentazione o il messaggio di errore. Una competenza chiave che i buoni programmatori hanno è la capacità di interpretare e seguire le istruzioni.",
+          ""
+        ],
+        [
+          "//i.imgur.com/GxvrsFb.gif",
+          "Una gif che mostra come fare una ricerca avanzata su Google. Prima di tutto, si inserisce la query \"jquery non viene eseguito quando la mia pagina si carica\". Poi si clicca sul pulsante strumenti di ricerca e si cambia l'opzione \"Qualsiasi data\" selezionando a \"Ultimo Anno\". Poi si clicca su un risultato, si legge l'articolo e troviamo la nostra risposta.",
+          "Se ciò non è stato d'aiuto, cerca su Google. Ci vuole un sacco di pratica per formulare delle buone query. Quando si esegue una ricerca su Google, di solito si preferisce includere il linguaggio o il framework che si sta utilizzando. Inoltre è meglio limitare i risultati ad un periodo recente . ",
+          ""
```

Comment:
@atjonathan I agree with @devilicecream. I was thinking the same thing while translating. What do you think about adding that to the italian version?